command/operator_diagnose.go

```diff
@@ -387,6 +387,57 @@ func (c *OperatorDiagnoseCommand) offlineDiagnostics(ctx context.Context) error
 		return nil
 	})
 
+	diagnose.Test(ctx, "seal-transit-tls-checks", func(ctx context.Context) error {
+		var checkSealTransit bool
+		for _, seal := range config.Seals {
+			if seal.Type == "transit" {
+				checkSealTransit = true
+
+				tlsSkipVerify, ok := seal.Config["tls_skip_verify"]
+				if !ok {
+					// Just warn here and not returning an error
+					diagnose.Warn(ctx, "Missing tls_skip_verify in the config")
```

Comment:
No need to warn if `tls_skip_verify` doesn't exist. In fact, in most cases it should not exist.

Reply:
Done